react/wbooks/src/app/screens/Home/layout.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
 import { array } from 'prop-types';
+import cn from 'classnames';
+import { t } from 'i18next';
 
 import NavBar from '../../components/NavBar';
 import BookInfo from '../../components/BookInfo';
 
 import styles from './styles.module.scss';
 
 function Home({ books }) {
+  const areBooks = books.length > 0;
```

Comment:
`const areBooks = !!books.length` as a suggestion, the variable name is weird, what about something like `booksAvailable`